webaudio-audionode-disconnect/index.html

```diff
@@ -0,0 +1,136 @@
+    <meta name="msapplication-TileImage" content="images/touch/ms-touch-icon-144x144-precomposed.png">
+    <meta name="msapplication-TileColor" content="#3372DF">
+
+    <link rel="icon" href="../images/favicon.ico">
+
+    <link rel="stylesheet" href="../styles/main.css">
+  </head>
+
+  <body>
+    <h1>WebAudio AudioNode selective disconnect Sample</h1>
+
+    <p>Available in <a href="http://www.chromestatus.com/feature/4793068276416512">Chrome 43+</a></p>
+
+    <p>
+      The overloaded method <code>AudioNode.disconnect</code> supports disconnecting either all connections, or a single connection (see the <a href="https://webaudio.github.io/web-audio-api/#idl-def-AudioNode">audio node spec</a>).
+      Selective disconnection, choosing a specific signal path to disconnect, is supported as of Chrome 43+.
+    </p>
+
+    <div class="output">
+      <div>
+        Plays a demo audio file, but only connects the right channel.
+        The left channel is toggled via <code>AudioNode.connect</code> and <code>AudioNode.disconnect</code> with selective arguments.
+      </div>
+      <button id="btn-play">Play</button>
+      <button id="btn-connect">Toggle Left Channel</button>
+      <pre id="result" style="color: red"></pre>
+    </div>
+
+    <!-- // [START code-block] -->
+    <script>
+
+    var context = new AudioContext();
+
+    // Load the audio file at the specified url, returning a Promise containing
+    // an AudioBuffer.
+    function loadAudio(url) {
+      // fetch is available in Chrome 42+.
+      return fetch(url).then(function(response) {
+        return response.arrayBuffer();
+      }).then(function(data) {
+        return new Promise(function(resolve, reject) {
+          context.decodeAudioData(data, function(buffer) {
+            buffer ? resolve(buffer) : reject();
+          });
+        });
+      })
+    }
+
+    var playerNode = context.createBufferSource();
+    playerNode.loop = true;
+
+    document.getElementById('btn-play').addEventListener('click', function() {
+      this.disabled = true;
+      playerNode.start();
+    });
+
+    loadAudio("../audio/techno.wav").then(function(buffer) {
+      playerNode.buffer = buffer;
+    }).catch(function(e) {
+      var result = document.getElementById('result');
+      result.textContent = 'Could not load demo audio: ' + e.toString();
+    });
+
+    // splitterNode splits the source audio into L/R channels.
+    var splitterNode = context.createChannelSplitter();
+    playerNode.connect(splitterNode);
+
+    // mergerNode merges arbitrary audio into L/R channels.
+    var mergerNode = context.createChannelMerger(2);
+    mergerNode.connect(context.destination);
+    splitterNode.connect(mergerNode, 0, 1); // flip left output to rightinput
+
+    var connected = false;
+    document.getElementById('btn-connect').addEventListener('click', function() {
+      // Connect/disconnect right output to left input.
+      if (connected) {
```

Comment:
Can you change the text in the button depending on the `connected` state? I think it would help users confirm that the demo is working if they saw something update each time they clicked. (The audio effect is kind of subtle, at least on my laptop speakers.)

Assigning to `this.textContent` might be sufficient, since you're already inside an event listener for the button.